app/jmeter/confluence.jmx

```diff
@@ -7836,13 +7841,13 @@ vars.put(&quot;p_new_file_name&quot;, (new Random().with {(1..9).collect {((&apo
                 <hashTree/>
                 <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="Response Assertion" enabled="true">
                   <collectionProp name="Asserion.test_strings">
-                    <stringProp name="175358039">title=&quot;Preview&quot;</stringProp>
+                    <stringProp name="2046823756">&lt;title&gt;Edit</stringProp>
```

Comment:
What does this name mean?

Reply:
I think this name="175358039" JMeter uses under the hood.